### ISSUE TITLE
Skip Capture With a Service

### ISSUE THREAD

**Issue by jrjohnson** [getsentry/sentry-symfony#188]
We'd like to be able to skip capture globally based on configuration. It seems like we have a few paths to do that right now.
1) Implement our own listener against the interface
2) Extend the existing listener and just override `shouldExceptionCaptureBeSkipped`

Both of these require writing a lot of boilerplate, would you be open to adding an autoconfigurable interface that would allow a service to respond to `shouldExceptionCaptureBeSkipped`?

I also tried to listen to the pre-capture event and stop it with `SentrySymfonyEvents::PRE_CAPTURE => 'onPreCapture'`

```php
public function onPreCapture(Event $event)
{
    if (!$this->errorCaptureEnabled) {
        $event->stopPropagation();
    }
}
```

While that method does get called, `stopPropagation` doesn't seem to stop the error from being sent to sentry.

**Comment by Jean85** [comment]
I'm currently working on a new major release in #184 to support the new 2.0 SDK. That version has many ways to dinamically stop sending events based on config, like a `before_send` callback that would allow you to intercept any event sent.

For the current version instead, if it's based on config, removing the DSN should do the trick.

**Comment by Jean85** [comment]
`before_send` callback has been implemented with #193. Expect a tagged beta version soon.
